scripts/godep-save

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+# make sure we are running in a toplevel directory
+if ! [[ -d '.git' ]]; then
+	echo "This script must be run in a toplevel rkt directory"
+	exit 255
+fi
+
+va=vendoredApps
+allApps=''
+if [[ -r "${va}" ]]; then
+	allApps=$(cat "${va}" \
+			 | grep -v '^[[:space:]]*\(//\|#\)' \
+			 | grep -v '^$' \
+			 | sed -e 's/[[:space:]]*\(#\|\/\/\).*//g')
+else
+	echo "No file '${va}' found"
+fi
+
+cmd="godep save -r ./... ${allApps}"
```

Comment:
For sanity, perhaps we should error out if godep is older than the version that has your patch..